src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,132 @@
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (mapIsLoad()) {
+            resize();
+        }
+        table = putElement(key, value, table);
     }
 
     @Override
     public V getValue(K key) {
+        Node<K, V> curentNode = table[getIndex(key)];
+        while (curentNode != null) {
+            if (keyCompare(curentNode.key, key)) {
+                return curentNode.value;
+            }
+            curentNode = curentNode.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void resize() {
+        Node<K, V>[] newTable = new Node[capasity << 1];
+        capasity = capasity << 1;
+        size = 0;
+        for (int i = 0; i < table.length; i++) {
+            if (table[i] == null) {
+                continue;
+            }
+            Node<K, V> curentNode = table[i];
+            while (curentNode != null) {
+                newTable = putElement(curentNode.key, curentNode.value, newTable);
+                curentNode = curentNode.next;
+            }
+        }
+        table = newTable;
+    }
+
+    private Node<K, V>[] putElement(K key, V value, Node<K, V>[] table) {
+        Node<K, V> newElement = new Node<>(key, value);
+        int index = getIndex(key);
+        if (table[index] != null) {
+            if (isKeyPresent(key, table)) {
+                changeElementInNodeList(table[index], newElement);
+                return table;
+            }
+            putToNodeList(table[index], newElement);
+            size++;
+            return table;
+        }
+        table[index] = newElement;
+        size++;
+        return table;
+    }
+
+    private void putToNodeList(Node<K, V> curentNode, Node<K, V> addNode) {
+        Node<K, V> node = curentNode;
+        while (node.next != null) {
+            node = node.next;
+        }
+        node.next = addNode;
+    }
+
+    private void changeElementInNodeList(Node<K, V> curentNode, Node<K, V> addNode) {
+        Node<K, V> node = curentNode;
+        while (node != null) {
+            if (keyCompare(node.key, addNode.key)) {
+                node.value = addNode.value;
+                return;
+            }
+            node = node.next;
+        }
+    }
+
+    private int getIndex(K key) {
+        return key == null ? 0 : Math.abs(key.hashCode() % capasity);
+    }
+
+    private boolean isKeyPresent(K key, Node<K, V>[] table) {
+        for (int i = 0; i < capasity; i++) {
+            if (table[i] != null) {
+                Node<K, V> curentNode = table[i];
+                while (curentNode != null) {
+                    if (keyCompare(curentNode.key, key)) {
+                        return true;
+                    }
+                    curentNode = curentNode.next;
+                }
+            }
+        }
+        return false;
+    }
+
+    private boolean keyCompare(K key1, K key2) {
```

Comment:
Using numbers in the variable names is bad practice. 
The method name should be a verb like `campareKeys` and similar